content/plugins/descriptions/travis-ci.yaml

```diff
@@ -19,24 +19,12 @@ coverImage:
   alt: 'A screenshot of the Travis CI plugin. It is showing a list of builds for a sample service.'
 
 gettingStarted:
-  - intro: 'Install the plugin in your Backstage instance'
-    language: 'bash'
-    code: 'yarn add @roadiehq/backstage-plugin-travis-ci'
-  - intro: 'Add the plugin to the list of plugins'
-    language: 'typescript'
-    code: |
-      // packages/app/src/plugins.ts
-      export { plugin as TravisCI } from '@roadiehq/backstage-plugin-travis-ci';
-  - intro: 'Add the API to your Backstage instance'
-    language: 'typescript'
-    code: |
-      // packages/app/src/api.ts
-      import { TravisCIApi, travisCIApiRef } from '@roadiehq/backstage-plugin-travis-ci';
-
-      // ...
-
-      builder.add(travisCIApiRef, new TravisCIApi());
```

Comment:
We should update this to be consistent with new way of adding plugins (plugins.ts)